presto-kudu/src/main/java/io/prestosql/plugin/kudu/KuduMetadata.java

```diff
@@ -394,7 +396,8 @@ public ConnectorTableProperties getTableProperties(ConnectorSession session, Con
                 handle.getTable(clientSession),
                 newDomain,
                 handle.getDesiredColumns(),
-                handle.isDeleteHandle());
+                handle.isDeleteHandle(),
+                handle.getLimit());
```

Comment:
if limit is present, we cannot apply filter, can we?

Reply:
Yes !! The `applyLimit` will be invoked only during this pattern `TableScan -> Limit` so in case of filters only if it is fully applied by the connector then limit might be pushed down

Reply:
Consider the cases:
```
Output -> Filter -> Limit -> Scan
```

and 
```
Output -> Limit -> Filter -> Scan
```

since we implement both applyLimit and applyFilter, how do we distinguish which one took place?

Reply:
I realized we had this discussion some time ago with JDBC connectors, but cannot find it now. @martint @electrum ?
Unfortunately we didn't capture much of the thinking in the code, and it _looks as if_ this was not addressed.